include/vapor/BarbParams.h

```diff
@@ -54,6 +57,8 @@ class PARAMS_API BarbParams : public RenderParams {
     //! \retval double line thickness
     double GetLineThickness() const { return GetValueDouble(_thicknessScaleTag, 1.f); }
 
+    //! Set line thickness in voxels
```

Comment:
Voxels? I don't think that is correct. The name of the tag, _thicknessScaleTag, would suggest that this a multiplier applied to the default line thickness

Reply:
Fixed.  I don't know where voxels came from.  I was just copying the documentation from the getter